python/semantic_kernel/connectors/ai/azure_ai_inference/services/azure_ai_inference_base.py

```diff
@@ -14,6 +16,14 @@
 class AzureAIInferenceBase(KernelBaseModel, ABC):
     """Azure AI Inference Chat Completion Service."""
 
+    # All Microsoft tools/SDKs that internally use the Azure AI Inference SDK are required to set
+    # a unique application ID, which will be part of the "use-agent" HTTP request header. This will
+    # allow teams to track usage from that tool/SDK, using service-side telemetry dashboards.
+    # Semantic Kernel's application ID is registered as "semantic-kernel-python/{version}".
+    # Note that the application ID is set only when a custom client is not provided, unless the
+    # custom client also sets the application ID.
+    _APPLICATION_ID: ClassVar[str] = APPLICATION_ID
```

Comment:
What's the advantage of using this class var and not just putting the const directly in the constructor?

Reply:
Good question! I am creating this ClassVar so that we can put a comment there documenting what this is and why we need it as this is AzureAIInference SDK specific.

Prior to the latest changes, the requested application ID was different than the one we already have thus the comment.

Reply:
I will remove this as we have agreed on using the existing one for Azure AI Inference.